bundle/run/job.go

```diff
@@ -317,6 +317,29 @@ func (r *jobRunner) Cancel(ctx context.Context) error {
 	return errGroup.Wait()
 }
 
+func (r *jobRunner) Restart(ctx context.Context, opts *Options) (output.RunOutput, error) {
+	// We don't need to cancel existing runs if the job is continuous and unpaused.
+	// the /jobs/run-now API will automatically cancel any existing runs before starting a new one.
+	//
+	// /jobs/run-now will not cancel existing runs if the job is continuous and paused.
```

Comment:
Did you mean "if the job is NOT continuous"?

Reply:
No, you can configure jobs to have continuous.pause_status = "PAUSED" in which case it behaves like a non-continuous job.